core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -293,11 +293,13 @@ public void applyAndJournal(Supplier<JournalContext> context, UpdateInodeFileEnt
    *
    * @param context journal context supplier
    * @param inode an inode to add and create a journal entry for
+   * @param path path of the new inode
```

Comment:
can `path` ever be `null` (optional)?

Reply:
It should not be `null` - All new inode entry (not from checkpoint) will contain a path.